src/redux/books/books.js

```diff
@@ -1,27 +1,95 @@
-// import { useId } from 'react-id-generator';
+// import Books from '../../components/Books';
```

Comment:
- [ ] Kindly remove the commented code to achieve a cleaner codebase

- [ ] Kindly add a description of the project to your README to make it more informative and descriptive.